bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -705,13 +712,25 @@ spec:
     type: ValidatingAdmissionWebhook
     webhookPath: /validate-mellanox-com-v1alpha1-nicclusterpolicy
   relatedImages:
-    - image: nvcr.io/nvidia/mellanox/mofed@sha256:ada9d1c4bf8eda0d6b76f2d3c0f67c9cd6862284f1312c6a6f142d5c217543b4
-      name: mofed
-    - image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin@sha256:f717f9778f48665b7c592f2225df51b755a1fe048125e034a286c564ee10fd37
+    - image: nvcr.io/nvidia/mellanox/doca-driver@sha256:3a42d15ff3ad5c8c7fe5b50cf78b2197cd9bb356c2df0db31ebc84b525d5e08b
```

Comment:
this one points to: nvcr.io/nvidia/mellanox/doca-driver:24.04-0.6.6.0-0-rhcos4.15

which is what we want :) for disconnected env if other image is needed then they can explicitly sync with oc mirror

Reply:
do we want to define other images with other names? e.g. doca-rhcos4.11?

Reply:
for now no IMO, part of @rollandf work to automate bundle builld will contain all relevant MOFED images of a given version.